app_streaming.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI
+from llama_cpp import Llama
+from transformers import pipeline
+from pydantic import BaseModel
+from fastapi.responses import StreamingResponse
+from typing import Optional
+
+# import uvicorn
+
+
+class Message(BaseModel):
+    text: str
+    temperature: Optional[float] = 0.3
+    max_tokens: Optional[float] = 100
+    top_p: Optional[float] = 0.1
+
+# echo = False
+# stop = ["Q", "\n"]
+
+
+instruct_model = Llama(model_path=r"<MODEL_PATH>")
+
+translation_model = pipeline("translation", model=r"<MODEL_PATH>", max_length=1000)
+app = FastAPI()
+
+
+@app.post("/summarize")
+async def summarize(message: Message):
+    translated_text = await translate(message.text, src_lang='heb_Hebr', tgt_lang='eng_Latn')
```

Comment:
Look at the comment below about the `translate` function - there is no need to `await` this function
```suggestion
    translated_text = translate(message.text, src_lang='heb_Hebr', tgt_lang='eng_Latn')
```